apps/daedong-user/src/components/ClubPost.tsx

```
@@ -0,0 +1,226 @@
+              <PostTitle>{postTitle}</PostTitle>
+            </Head>
+            <CategoryContainer>
+              {categories.map((categorie, index) => (
+                <Category key={index}>{categorie}</Category>
+              ))}
+            </CategoryContainer>
+            <Deadline>지원 마감일 : {deadline}</Deadline>
+          </ContentContainer>
+          <ButtonWrapper>
+            <Button
+              children={expanded ? '상세보기 접기' : '상세보기'}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)